stylesheets/components/text-utils.css

```diff
@@ -0,0 +1,19 @@
+.grey-text{
+    color: grey;
```

Comment:
you can add "grey" color in  variable and use variable instead

Reply:
Updated!! please check & approve.

Reply:
You can merge